virtool/groups/oas.py

```diff
@@ -79,8 +79,14 @@ class EditGroupSchema(BaseModel):
         default={},
     )
 
+    name: Optional[constr(min_length=1)] = Field(
+        description="a name for the group", default=None
+    )
+
     class Config:
-        schema_extra = {"example": {"permissions": {"create_ref": True}}}
+        schema_extra = {
+            "example": {"permissions": {"create_ref": True}, "name": "Gobblers"}
```

Comment:
This is not appropriate for public API documentation... 🤔
```suggestion
            "example": {"permissions": {"create_ref": True}, "name": "Technicians"}
```